apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -0,0 +1,264 @@
+import { View, Text, ScrollView, StyleSheet } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import {
+  Card,
+  IconButton,
+  Divider,
+  Paragraph,
+  TextInput
+} from 'react-native-paper';
+import BigButton from '../components/BigButton';
+import { colors } from '../config/colors';
+
+export const NewQuizScreen = () => {
+  const [text, onChangeText] = useState('');
+  const [numLines, changeNumLines] = useState(1);
+
+  useEffect(() => {
+    const charLen = text.length;
+    const suggestLines = Math.ceil(charLen / 50);
+    suggestLines > 0 ? changeNumLines(suggestLines) : changeNumLines(1);
+  }, [text]);
+
+  const [numQuestions, setNumQuestions] = useState(1);
+
+  const handleIncrement = () => {
+    setNumQuestions(numQuestions + 1);
+  };
+
+  const handleDecrement = () => {
+    if (numQuestions > 1) {
+      setNumQuestions(numQuestions - 1);
+    }
+  };
+
+  return (
+    <>
+      <ScrollView>
+        <View style={{ flex: 1, marginHorizontal: 35 }}>
+          <View>
+            <Text style={localStyles.pageTitle}>Create New Quiz</Text>
+          </View>
+          <Text style={localStyles.title}>Source Material</Text>
+          <View style={localStyles.container}>
+            <TextInput
```

Comment:
Could you use menuinput here or not? I prefer the lightblue text moving on edge. 
For color choose, white and grey are not contrasting color.

Reply:
https://discord.com/channels/1083967031893831770/1083967032346804249/1127429110596108319

Reply:
![image](https://github.com/new-idea-machine/os-lesson-delivery-system/assets/105611260/d7b1d4c8-7cf4-434b-821e-8898eeb11f50)

Reply:
I didn't use menuinput because there are some built-in features I am not sure it is needed.

Reply:
I am not sure this too, I feel white text is not clear. If you want to make 'pretty' moving, menuinput is better. 

But it depends on you.